src/editor_view.cpp

```diff
@@ -242,7 +244,22 @@ void EditorView::mouseMoveEvent(QMouseEvent * event)
             if (m_mediator.selectionGroupSize()) {
                 m_mediator.moveSelectionGroup(*node, m_grid.snapToGrid(m_mappedPos - m_mediator.mouseAction().sourcePosOnNode()));
             } else {
-                node->setLocation(m_grid.snapToGrid(m_mappedPos - m_mediator.mouseAction().sourcePosOnNode()));
+                auto cur_pos = m_mappedPos - m_mediator.mouseAction().sourcePosOnNode();
```

Comment:
Variable names not according to the style. Local variables should be `camelCase`'d instead of `snake_case`. This is mentioned in `CONTRIBUTING.md`.